Repositories.Entities/Package.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Repositories.Entities
+{
+    public class Package
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string? _id { get; set; }
+
+        [BsonElement]
+        public string Name { get; set; }
+
+        [BsonElement]
+        public string Description { get; set; }
+
+        [BsonElement]
+        public PostMachine SourceMachine { get; set; }
+
+        [BsonElement]
+        public PostMachine DestinationMachine { get; set; }
```

Comment:
I think we need not just object ids but relation ids for entity framework to understand. For example here we would need DestinationMachine id or else entity framework .Include would not work. I am not 100% sure though, will need to test this out.